src/Symfony/Component/Notifier/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * Added `DSN::getRequiredOption` method which throws a new `RequiredOptionMissingException`.
+   This new exception extends `IncompleteDsnException` to stay BC.
```

Comment:
not needed in the changelog IMHO

Reply:
You mean the second sentence right?

Reply:
Yes, providing a BC Layer is kind of "expected", no need to add a sentence about this in the changelog